File: solutions/uri/1006/1006.cs
using System;
using System.Collections.Generic;

class Solution {
    static void Main() {
        double a, b, c;
        string val;

        val = Console.ReadLine();
        a = Convert.ToDouble(val);
        val = Console.ReadLine();
        b = Convert.ToDouble(val);
        val = Console.ReadLine();
        c = Convert.ToDouble(val);

        Console.Write(
            "MEDIA = {0:F1}{1}",
            (a * 2.0 + b * 3.0 + c * 5.0) / 10.0,
            Environment.NewLine
        );
    }
}

File: solutions/uri/1006/1006.hs
import Text.Printf

main :: IO ()
main = do
    a <- readLn :: IO Float
    b <- readLn :: IO Float
    c <- readLn :: IO Float
    printf "MEDIA = %.1f\n" ((a * 2.0 + b * 3.0 + c * 5.0) / 10.0)
